src/openforms/submissions/models/submission.py

```diff
@@ -642,3 +649,13 @@ def get_prefilled_data(self):
                 variable.key: variable.value for variable in prefill_vars
             }
         return self._prefilled_data
+
+    def get_registration_plugin(self):
+        from ..tasks.registration import get_registration_plugin
+
+        return get_registration_plugin(self)
```

Comment:
I don't like this - celery tasks should really be leaf nodes that use other utilities and not the other way. additionally, you're now leaking registration app/module details in the core submissions app: see https://open-forms.readthedocs.io/en/latest/developers/architecture.html#core

I'd move this `get_registration_plugin` utility into `openforms.registrations.service` instead